<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domain</class>
 <widget class="QDialog" name="domain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>295</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>현대하모니 M</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CCUI</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>Doamin</string>
   </property>
  </widget>
  <widget class="QComboBox" name="domainBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>COMMON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SD_CARD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HELP_PHONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HELP_RADIO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HELP_NAVI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HELP_UVO</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="language">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </widget>
  <widget class="QComboBox" name="langBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>KOK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ENU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FRC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SPM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ENG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DUN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FRF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SPE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ITI</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="platform">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>Platform</string>
   </property>
  </widget>
  <widget class="QComboBox" name="platformBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>AVN5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVN5(18MY)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVN5(19MY)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WIDE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WIDE(18MY)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WIDE(19MY)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRE5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRE6</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="useclc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>UseCLC</string>
   </property>
  </widget>
  <widget class="QComboBox" name="useclcBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>ACMOD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="acmodText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>acmod5_4000_eng_gen_car_f16_v2_0_0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>CLC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clcText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>clc_eng_cfg3_v6_1_4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="horizontalBar_line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="tool">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>Tool</string>
   </property>
  </widget>
  <widget class="QComboBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>4.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.11</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="allStartButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>전체 실행</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fsrCheck">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>fsr 생성하지 않음</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="paramCheck">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>param 생성하지 않음</string>
   </property>
  </widget>
  <widget class="QPushButton" name="indeStartButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>현재 Domain만 실행</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
    </font>
   </property>
   <property name="text">
    <string>결과 폴더 열기</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Additional Option</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wideButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>16</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>현대하모니 M</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>allStartButton</sender>
   <signal>clicked()</signal>
   <receiver>domain</receiver>
   <slot>slot_all_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>domain</receiver>
   <slot>exec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indeStartButton</sender>
   <signal>clicked()</signal>
   <receiver>domain</receiver>
   <slot>slot_inde_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>domain</receiver>
   <slot>slot_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wideButton</sender>
   <signal>clicked()</signal>
   <receiver>domain</receiver>
   <slot>slot_wide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_lineEdit()</slot>
  <slot>slot_all_start()</slot>
  <slot>slot_inde_start()</slot>
  <slot>slot_open()</slot>
  <slot>slot_wide()</slot>
 </slots>
</ui>
